packages/teleport/src/features.ts

```diff
@@ -347,6 +348,32 @@ export class FeatureApps {
   }
 }
 
+export class FeatureKubes {
+  getTopNavTitle() {
+    return 'Kubernetes';
+  }
+
+  route = {
+    title: 'Kubernetes',
+    path: cfg.routes.kubernetes,
+    exact: true,
+    component: Kubes,
+  };
+
+  register(ctx: Ctx) {
```

Comment:
kube feature is similar to apps where it can be enabled/disabled depending on the users RBAC rules. Their role can say `disable kube feature`, then we shouldn't allow user to access this feature on UI. Examine how Apps is being registered

Reply:
implemented feature flag for kube server that allows depending on users RBAC rules to toggle for list, read, edit, create, and remove options.